### ISSUE TITLE
Bug: In user.cfg kernel src path defined but defconfig files not showing properly.

### ISSUE THREAD

**Issue by mmtrt** [grm34/ZenMaxBuilder#754]
when manually adding kernel src path on prompt it is showing all present defconfigs properly but when its defined in user.cfg then its showing like this.

```
1 *_defconfig
2 vendor/*_defconfig
```

**Comment by grm34** [comment]
What's the output of

`uname -a && bash --version | grep bash && ps -p $$`

**Comment by mmtrt** [comment]
```
Linux kubuntu 6.0.6-x64v2-xanmod1 #0~20221029.150b113 SMP PREEMPT_DYNAMIC Sat Oct 29 19:07:24 UTC  x86_64 x86_64 x86_64 GNU/Linux
GNU bash, version 5.2.2(1)-release (x86_64-pc-linux-gnu)
    PID TTY          TIME CMD
 383032 pts/1    00:00:00 bash
```

**Comment by grm34** [comment]
> ```
> Linux kubuntu 6.0.6-x64v2-xanmod1 #0~20221029.150b113 SMP PREEMPT_DYNAMIC Sat Oct 29 19:07:24 UTC  x86_64 x86_64 x86_64 GNU/Linux
> GNU bash, version 5.2.2(1)-release (x86_64-pc-linux-gnu)
>     PID TTY          TIME CMD
>  383032 pts/1    00:00:00 bash
> ```

Thank you.
I can reproduce, I'm looking on after work.

**Comment by grm34** [comment]
@mmtrt fixed by #755 & #757 (happened if you added a `/` at the end of the kernel path).

Thanks for reporting 🙏
